ui/lib/open-api-explorer/addon/routes/index.js

```diff
@@ -17,7 +17,6 @@ IF YOUR TOKEN HAS THE PROPER CAPABILITIES, THIS WILL CREATE AND DELETE ITEMS ON
 Your token will also be shown on the screen in the example curl command output.`;
     this.flashMessages.warning(warning, {
       sticky: true,
-      preformatted: true,
```

Comment:
The new component renders this correctly by default: 
![Uploading Screenshot 2024-02-15 at 13.59.29.png…]()